recipes/sentry-crashpad/all/conandata.yml

```diff
@@ -0,0 +1,32 @@
+sources:
+  "0.4.9":
```

Comment:
Why pulling `sentry-native` instead of just the submodule?
https://github.com/getsentry/crashpad
Is it because of the tags/releases?

Reply:
That repo does no releases and has no stable branches.
They rebase on top of google crashpad's master.